### ISSUE TITLE
Audience claim verification for Access Tokens should allow an array of strings

### ISSUE THREAD

**Issue by AbelVandenBriel** [solid/identity-token-verifier#6]
The [Solid-OIDC spec](https://solid.github.io/authentication-panel/solid-oidc/#tokens) prescribes that "The audience claim MUST either be the string solid or be an array of values, one of which is the string solid".

[Section 4.1.3](https://tools.ietf.org/html/rfc7519#section-4.1.3) of the JSON Web Token spec also echoes that the aud claim "is an array of case-sensitive strings" or just a singular string.

However, the CSS currently errors when presented with an audience claim that is an array of strings.

The aud claim in the Access Token:
```
"aud": [
    "http://example.com",
    "solid"
  ]
```

The CSS error:
```
BadRequestHttpError: Error verifying WebID via DPoP-bound access token: Expected solid, got:
http://example.com,solid
    at DPoPWebIdExtractor.handle (C:\Users\Abel\Documents\Digita\repos\community-server\dist\authentication\DPoPWebIdExtractor.js:50:19)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async AuthenticatedLdpHandler.runHandlers (C:\Users\Abel\Documents\Digita\repos\community-server\dist\ldp\AuthenticatedLdpHandler.js:69:29)
    at async AuthenticatedLdpHandler.handle (C:\Users\Abel\Documents\Digita\repos\community-server\dist\ldp\AuthenticatedLdpHandler.js:46:61)
    at async SequenceHandler.handle (C:\Users\Abel\Documents\Digita\repos\community-server\dist\util\handlers\SequenceHandler.js:27:26)
    at async Server.<anonymous> (C:\Users\Abel\Documents\Digita\repos\community-server\dist\server\BaseHttpServerFactory.js:25:17)
```

**Comment by matthieubosquet** [comment]
It was fixed in v0.6.3. I thought it would come through since it's a patch.

I updated all dependencies of @solid/identity-token-verifier, published a new minor. Pull request for updated dependency in the solid community server is here: https://github.com/solid/community-server/pull/708

See also: https://github.com/solid/identity-token-verifier/pull/5/files & https://github.com/solid/identity-token-verifier/issues/3

**Comment by joachimvh** [comment]
> It was fixed in v0.6.3. I thought it would come through since it's a patch.

The `package-lock.json` prevents this from automatically being used.

**Comment by matthieubosquet** [comment]
That makes sense!
